### PR TITLE
Webgl communication bundle refactor

### PR DESCRIPTION
**Pull request by Ysabz**
This branch adds the pluging for the webgl browser communication. Also, a new scene and container code are created that loads the apparatus as asset bundles from azure instead of addressables . 
To test: 

 build the scene appratuscontainer_exampleassetbundlecloud and try the load and trigger buttons. the assets are all loaded from the azure . 

To test webgl/browser communication : 

after building the scene, got to index.html and add this to store the unity instance in uin variable : 
 var uin = null;
  script.onload = () => {
    createUnityInstance(canvas, config, (progress) => {...}).then((unityInstance) => {
      uin = unityInstance;
      …

Then click on cube button to load the apparatus. Then in the browser console , type this command :  uin.SendMessage("Container",”VoidTrigger", "gracious-cube/gracious-cube/delta-sphere/wobble-sphere/wobble-sphere/animations@Wobble") to make the spehere wobble. 

A message should be printed on the browser console : " The void trigger is handled successfully"

### CODE REVIEW

File: Assets/Scripts/ApparatusContainer_ExampleCloudBundles.cs [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+
+namespace Atomata.VSolar.Apparatus.Example
+{
+    /// <summary>
+    /// This is an example container, that has only been created to show quickly how
+    /// a container can load and unload an apparatus from byte[] data. The byte data in
+    /// this case is stored on the filesystem of the user in a specific place. The container
+    /// is useless if the database hasn't been prepared manually.
+    /// </summary>
+    public class ApparatusContainer_ExampleCloudBundles : MonoBehaviour
```
Comment (hexthedev) [inline Assets/Scripts/ApparatusContainer_ExampleCloudBundles.cs:24]: Make sure to update the summary

**Review by hexthedev — APPROVED** [review]
Good job Yasaman. This is exactly what we needed next.

File: Assets/Scripts/ApparatusContainer_ExampleCloudBundles.cs (hunk 1 above)
Comment (Ysabz) [reply Assets/Scripts/ApparatusContainer_ExampleCloudBundles.cs:24]: Sure, I'll fix it

Comment (Ysabz) [reply Assets/Scripts/ApparatusContainer_ExampleCloudBundles.cs]: fixed now.

**Pull request merged**